test_get_health test need adjustment for Provider mode platform

https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#ocs/launches/795/28001/1375360/1375555/log

self = <Popen: returncode: -9 args: ['odf', '-n', 'openshift-storage-client', 'get'...>
endtime = 227123.268550651, orig_timeout = 600, stdout_seq = [b'\n']
stderr_seq = [b'Info: ', b'waiting for pod with label "app=rook-ceph-operator" in namespace "openshift-storage-client" to be runnin...nfo: waiting for pod with label "app=rook-ceph-operator" in namespace "openshift-storage-client" to be running\n', ...]
skip_check_and_raise = False


def _check_timeout(self, endtime, orig_timeout, stdout_seq, stderr_seq,
                   skip_check_and_raise=False):
    """Convenience for checking if a timeout has expired."""
    if endtime is None:
        return
    if skip_check_and_raise or _time() > endtime:

      raise TimeoutExpired(
                self.args, orig_timeout,
                output=b''.join(stdout_seq) if stdout_seq else None,
                stderr=b''.join(stderr_seq) if stderr_seq else None)
E           subprocess.TimeoutExpired: Command '['odf', '-n', 'openshift-storage-client', 'get', 'health']' timed out after 600 seconds



/usr/lib64/python3.9/subprocess.py:1178: TimeoutExpired

--- a/tests/functional/odf-cli/test_get_commands.py
+++ b/tests/functional/odf-cli/test_get_commands.py
@@ -9,6 +9,7 @@ from ocs_ci.framework.testlib import (
     polarion_id,
     skipif_ocs_version,
     skipif_external_mode,
+    runs_on_provider,
 )
 
 log = logging.getLogger(__name__)
@@ -23,6 +24,7 @@ class TestGetCommands:
         self.odf_cli_runner = odf_cli_setup
 
     @skipif_external_mode
+    @runs_on_provider
     @polarion_id("OCS-6237")
     def test_get_health(self):
         output = self.odf_cli_runner.run_get_health()

